src/components/basic/Menu/index.tsx

```diff
@@ -40,25 +42,66 @@ export const Menu = ({
   component,
   onClick,
   notificationInfo,
+  activePathname,
+  subMenuDivider,
   ...props
 }: MenuProps) => {
   const { spacing } = useTheme()
+  const currentPath = window.location.pathname
 
   return (
-    <Box {...props}>
-      <List sx={{ padding: 0 }}>
-        {items?.map((item) => (
-          <MenuItem
-            {...item}
-            component={component}
-            menuProps={props}
-            Menu={Menu}
-            onClick={onClick}
-            key={uniqueId('Menu')}
-            showNotificationCount={item.to === 'notifications'}
-            notificationInfo={notificationInfo}
-          />
-        ))}
+    <Box {...props} className="cx-menu">
+      <List sx={{ padding: 0 }} className="cx-menu__list">
+        {items?.map((item, index) => {
+          if (item.children) {
+            return (
+              <>
+                <MenuItem
+                  title={item.title}
+                  key={`header-${index}`}
```

Comment:
I would avoid using array indices as keys because they can cause issues with React's reconciliation process, especially when items are reordered or deleted.

```suggestion
                    key={item.to ?? item.href ?? `menu-${item.title}`}
```